baseapp-profiles/baseapp_profiles/signals.py

```diff
@@ -1,39 +1,28 @@
 import swapper
-from django.apps import apps
 from django.contrib.contenttypes.models import ContentType
 
-if apps.is_installed("avatar"):
-    from avatar.models import Avatar
-else:
-    Avatar = None
-
 
 def update_user_profile(instance, **kwargs):
+    if not hasattr(instance, "profile"):
+        return
```

Comment:
could we raise an exception instead?

maybe some

`except AttributeError`

Reply:
@vitorguima I can't remember why I added this condition, can you believe? I will just remove, `instance.profile` is not even used in that function.